src/qt3utils/datagenerators/princeton.py

```diff
@@ -0,0 +1,339 @@
+import os
+import numpy as np
+from time import sleep
+
+try:
+    import clr
```

Comment:
what is 'clr' and how is it installed?

Reply:
It is there so that I can interact with python.NET. I am unable to interact with the Lightfield software without this import. It was also listed as "required" in the Lightfield documentation. It is installed by using **pip install pythonnet**